apps/dashboard/src/lib/components/Badge/index.svelte

```diff
@@ -0,0 +1,14 @@
+<script>
+</script>
+
+<div>
+  <span
+    class="flex items-center gap-2 dark:text-black dark:bg-white bg-gray-100 rounded-md p-3 font-semibold shadow-md"
```

Comment:
1. Remove the shadow completely
2. Change it to 
```
<logo> Powered by ClassroomIO
```
Note: logo should be `/logo-192`, cause I can see from your screenshot that the quality of the logo is low

Reply:
3. Remove the bg-gray and make it border-gray